docs/snippets/ov_extensions.py

```diff
@@ -2,10 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-import openvino.runtime as ov
+import openvino as ov
 
 #! [py_frontend_extension_ThresholdedReLU_header]
-import openvino.runtime.opset8 as ops
```

Comment:
It needs to be done this way. Opsets are not exposed into top-level namespace.

Reply:
Have changed it in the https://github.com/openvinotoolkit/openvino/pull/19084/commits/5b7da5dc2ee73f1c39be7215b6771c39830be92b commit.